handbook/ops/finance/arr.md

```diff
@@ -0,0 +1,46 @@
+# ARR
+
+## Definitions
+
+See [definitions of ARR, IARR, new IARR, expansion IARR, and bookings](../../sales/index.md#definitions) in the sales handbook.
+
+## Monthly closing process
+
+Every month the finance team closes the books and reports the latest ARR to the team based on calculations in the ARR source of truth, the [ARR tracker spreadsheet](https://docs.google.com/spreadsheets/d/1Ao3Nqw6gH3yAuZtICV3xo35kKKnI9oKXnvPuTQ0Fh9c/edit). This process will be completed within 5 business days of the end of the month.
+
+### New IARR
+
+1. Throughout the month, finance will take any announced sales and add them to the ARR tracker spreadsheet on the Deals tab. Whenever a sale is announced in the #sales Slack channel, before adding it to the ARR tracker spreadsheet, finance will respond to the AE with their assumptions for the following values, and sales must confirm in writing that each of these is correct:
+  - Customer name
+  - Close date
+  - IARR amount
+
+### Expansion IARR
+
+1. At the end of the month, finance will calculate [expansion IARR and MAU bookings](https://docs.google.com/spreadsheets/d/1tRcz3bNOho1TyWvrYSv37RIYcQs7I0i05-5eKwLq8TI/edit#gid=0). Finance will post a message in the #sales channel listing each individual booking, with the AE who owns the account tagged on each line. Each AE with a booking must confirm in writing that the expansion is correct.
```

Comment:
I love this, would make me feel better about the expansion process 😄. For now I agree this should be in Slack but should migrate over to Salesforce if/when we do expansion tracking there.